anvio/drivers/hmmer.py

```diff
@@ -203,20 +203,29 @@ def run_hmmer(self, source, alphabet, context, kind, domain, num_genes_in_model,
             output_file = partial_input_file + '_output'
             table_file = partial_input_file + '_table'
 
+            if domtblout_path:
+                domain_table_file = domtblout_path + '_domtable'
+            else:
+                domain_table_file = partial_input_file + '_domtable'
+
             self.run.info('Log file for thread %s' % thread_num, log_file)
             thread_num += 1
 
+            domain_out_fmt = "--domtblout"
+
             if noise_cutoff_terms:
                 cmd_line = ['nhmmscan' if alphabet in ['DNA', 'RNA'] else self.program_to_use,
                             '-o', output_file, *noise_cutoff_terms.split(),
                             '--cpu', cores_per_process,
                             out_fmt, table_file,
+                            domain_out_fmt, domain_table_file,
```

Comment:
Does this not fail when `domtblout_path` is None? I would expect it to fail in all the other conditionals in which `cmd_line` is defined, too.

Reply:
I tried to address this [here](https://github.com/merenlab/anvio/pull/1678/commits/fb28845c4b873155423c677f9423ec2eadc44a8b)